src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/ControllerAnnotations.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
+annotation class ProtectedByIngress
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
+annotation class PublicEndpoint
```

Comment:
Annotations that can be used to denote that an endpoint is protected by other means or explicitly state that its entirely open